### PR TITLE
Temporary PR to review some possible extra criteria

### PR DESCRIPTION
**Pull request by Luca-Terrazzan**
Review from a [Discord thread](https://discord.com/channels/811981269595324436/1152272303103684680/1152272303103684680)

### CODE REVIEW

File: src/soft-play.js [hunk 1]
```diff
@@ -6,9 +6,62 @@ let children = 0
 // the adults and children variables defined above.
 // Start with the occupancy function.
 
+// here is what the object of the softballCenter looks like
+new softballCenter = (sport) = {
```
Comment (Luca-Terrazzan) [inline src/soft-play.js:10]: Have you tried to run either this script or tests?

File: src/multipleCenters.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+// array of all centers
+const centers = []
```
Comment (Luca-Terrazzan) [inline src/multipleCenters.js:2]: I think the approach of storing centers in a list is a fine idea for this exercise; later on, you will have a few more tools that will allow you to model this extra requirement a lot better!

File: src/multipleCenters.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    adults: 0,
+    children: 0
+  }
+  this.totalTally = {
+    adults: 0,
+    children: 0
+  }
+}
+
+const addCenter = (sport) => centers.push(new center(sport))
```
Comment (Luca-Terrazzan) [inline src/multipleCenters.js:40]: This usage of the `new` keyword works as expected and is fine to create a new `instance` of a center; later on, you might discover that JS has a way to represent instances of objects with the concept of `class`, which would be very appropriate in this situation. You are very welcome to explore that if you want to get a taste of object oriented programming :) Keep in mind that it's all syntactical sugar, which means that at the end of the day, a class is just a fancier way to represent an object/function

File: src/soft-play.js (hunk 1 above)
Comment (extra-jalapenos) [reply src/soft-play.js:10]: Whoops, I think that's a leftover change before I realized I probably should move this to a different file to not totally destroy the predefined tests.

**Pull request closed without merging**